doc/chapters/api/index.rst

```diff
@@ -32,102 +32,187 @@ BindMount
 Binds a path on the host to the container. Any missing parent directories in
 the path will be created.
 
-:Parameters:
-        :containerID: ``int32`` The ID obtained by CreateContainer method.
-        :pathInHost: ``string`` absolute path in the host.
-        :pathInContainer: ``string`` the absolute path to mount to in the container.
-        :readOnly: ``bool`` indicates whether the directory is read-only or not.
-
-|
+Parameters
+##########
+* containerID: ``int32`` The ID obtained by CreateContainer method.
+* pathInHost: ``string`` absolute path in the host.
+* pathInContainer: ``string`` the absolute path to mount to in the container.
+* readOnly: ``bool`` indicates whether the directory is read-only or not.
+
+Error sources
```

Comment:
I don't think this chapter reads nicely. The opening paragraphs are a bit off, they look like each sentence was added by a different person. But more to the point is that they don't really provide the information needed to understand e.g. what an "error source" is. I'm guessing it's "a reason something failed"? Is "Bad state" a special case? It looks  like it since it's followed by a colon and the other items in the list have sub-items. Is there a connection between these items and the error message the user receives? I'm not saying that the message should be duplicated here (maintenance problem), but there should be some hint of how to connect an "error source" to the user experience.